docs/administration/Configuration.md

```diff
@@ -509,6 +509,11 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - **Default**: 10000
 - **Description**: The timeout duration for committing (publishing) a write transaction to a StarRocks external table. The default value `10000` indicates a 10-second timeout duration.
 
+##### enable_sync_publish
+
+- **Default**: true
+- **Description**: Whether to synchronously execute the apply task at the publish phase of a load transaction. This parameter is applicable only to Primary Key tables. Valid values: <ul><li>`true`: The apply task is synchronously executed at the publish phase of a load transaction. It means that load transaction is reported successful only after apply task is completed, and the loaded data can truly be queried. It can improve query performance and stability when frequently load large batches of data into Primary Key tables but may significantly increase load latency.</li><li>`false`: The apply task is asynchronously executed at the publish phase of a load transaction. It means that load transaction is reported successful after the apply task is  submitted, but the loaded data can not be immediately queried. In this case, concurrent queries need to wait for the apply task to complete or timeout before they can continue, which may affect query performance and stability.</li></ul>
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/417ccb7e-1de1-4de0-ad65-725d3cfb306b)
"true" is the default value. Point that out in the parameter description.